docs/modules/operation/pages/deep-dive/events/event-tokens.adoc

```diff
@@ -1,121 +1,174 @@
-If a literal % sign appears in your description, you must escape it with a %.
-For example,
-
-`'testUrl \https://www.opennms.org/ReportSection?filter=FOO_BAR_BAZ%%20eq%%20%%27101%%27'`
-
-%eventid%::
-    The event's numeric database ID.
-%uei%::
-    The unique event identifier for the event.
-%source%::
-    The source of the event (which {page-component-title} service daemon created it).
-%descr%::
-    The event description.
-%logmsg%::
-    The event logmsg.
-%time%::
-    The time of the event.
-%shorttime%::
-    The time of the event formatted using `DateFormat.SHORT` for a completely numeric date/time.
-%dpname%::
-    Formerly used for the ID of the Minion that the event was received on.
-    This placeholder is deprecated and resolves to an empty String.
-%nodeid%::
-    The numeric node ID of the device that caused the event, if any.
-%nodelabel%::
-    The node label for the node given in `%nodeid%`, if available.
-%nodelocation%::
-    The node location for the node given in `%nodeid%`, if available.
-%host%::
-    The host at which the event was generated.
-%interface%::
-    The IP interface associated with the event, if any.
-%foreignsource%::
-    The requisition name for the node given in `%nodeid`, if available.
-%foreignid%::
-    The requisition ID for the node given in `%nodeid`, if available.
-%ifindex%::
-    The interface's SNMP ifIndex.
-%interfaceresolve%::
-    Does a reverse lookup on the `%interface%` and returns its name, if available.
-%service%::
-    The service associated with the event, if any.
-%severity%::
-    The severity of the event.
-%snmphost%::
-    The host of the SNMP agent that generated the event.
-%id%::
-    The SNMP enterprise OID for the event.
-%trapoid%::
-    Full trap OID for the event.
-%idtext%::
-    The decoded (human-readable) SNMP Enterprise OID for the event.
-%ifalias%::
-    The interface's SNMP ifAlias.
-%generic%::
-    The generic trap-type number for the event.
-%specific%::
-    The specific trap-type number for the event.
-%community%::
-    The community String for the trap.
-%version%::
-    The SNMP version of the trap.
-%snmp%::
-    The SNMP information associated with the event.
-%operinstruct%::
-    The operator instructions for the event.
-%mouseovertext%::
-    The mouse-over text for the event.
-%tticketid%::
-    The trouble ticket ID associated with the event, if available.
-%primaryinterface%::
-    The primary interface IP address for the node given in `%nodeid%`, if available.
-
-CAUTION: The use of multiple Minions in one location can break the alarm life cycle for some {page-component-title} features.
-To avoid this problem, replace the `%dpname%` with an empty String by setting `org.opennms.netmgt.eventd.cleardpname=true` in `$\{OPENNMS_HOME}/etc/opennms.properties.d/events.properties`.
+You can use replacement tokens (event substitution tokens) in event descriptions, log messages, operator instructions, and automatic actions.
```

Comment:
I feel like the only thing that is missing here is something that defines what a replacement token (or a token) is.  I have an understanding from what is here but something like "A replacement token is a way to substitute additional relevant data into an event" or something... I don't know what the exact definition should be but I think it would help.

Reply:
We could modify the introductory text to be more specific with the straightforward definition and then the more detailed info about the process.

Reply:
```suggestion
You can use replacement tokens (event substitution tokens) to insert additional relevant information into event descriptions, log messages, operator instructions, and automatic actions.
```